### PR TITLE
Move search tab pagination

### PR DESCRIPTION
**Pull request by ambuznego**

### CODE REVIEW

File: assets/js/components/common/Mapbox.jsx [hunk 1]
```diff
@@ -35,7 +35,7 @@ export default (props) => {
           alias: hotspot_alias,
           packetCount: h.packet_count,
           isFollowed: String(
-            props.followedHotspotStats.filter(function (followedHotspot) {
+            props.allOrganizationHotspots.filter(function (followedHotspot) {
```
Comment (vicmgs) [inline assets/js/components/common/Mapbox.jsx:38]: could you use orgHotspotMap to find if these hotspots are followed? orgHotspotsMap is just the allOrgHotspots array converted to a map

File: assets/js/components/coverage/CoverageIndex.jsx [hunk 2]
```diff
@@ -212,27 +124,27 @@ export default (props) => {
...
           );
         }
       }
     } else if (currentTab === "followed") {
-      if (followedHotspotStatsData) {
+      if (allOrganizationHotspotsData) {
         if (hotspotAddressSelected) {
           const selectedHotspot =
-            followedHotspotStatsData.followedHotspotStats.filter(
+            allOrganizationHotspotsData.allOrganizationHotspots.filter(
```
Comment (vicmgs) [inline assets/js/components/coverage/CoverageIndex.jsx:139]: same here

**Review by vicmgs — COMMENTED** [review]
just one comment

File: assets/js/components/coverage/CoverageIndex.jsx (hunk 2 above)
Comment (ambuznego) [reply assets/js/components/coverage/CoverageIndex.jsx:139]: on this one, I can't bc i need more data than what's in `orgHotspotMap`, i.e. coordinates

Comment (ambuznego) [reply assets/js/components/coverage/CoverageIndex.jsx:139]: i just realized it's broken even w/ `allOrganizationHotspotsData`

Comment (vicmgs) [reply assets/js/components/coverage/CoverageIndex.jsx:139]: got it, feel free to merge

**Pull request merged**